contracts/Oracle/Calculations/Sushiswap.sol

```diff
@@ -152,21 +142,6 @@ contract CalculationsSushiswap {
         }
     }
 
-    function getRouterForLpToken(address tokenAddress)
```

Comment:
Remove unused legacy method that relies on specifying factory address. This method is not used in all of lens